### PR TITLE
Created setup Batch script for Windows

### PR DESCRIPTION
**Pull request by bartekpacia**
### Description
So, [having created this in Bash](https://github.com/systers/mentorship-backend/pull/270), I thought "let's try on Windows now" and I think I might regret my decision. Batch files are a nightmare, literally *horrible*. Nevertheless I created this simple, dummy so-to-say script.

~~However, it *just stops*  after line 40 (in other words, after printing `virtual environment created`.
If anybody could guide me a bit why it doesn't work, I'd be very grateful.~~ fixed

### Type of Change:

- Code

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Manually a few times and...works!

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

PS I'm not sure whether it's a right thing to do (to create a PR when functionality isn't finished yet). If it's wrong, sorry in advance.

### CODE REVIEW

File: install.bat [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+@echo set APP_MAIL_USERNAME=%username%>> .env
+@echo set APP_MAIL_PASSWORD=%password%>> .env
+
+pip3 install virtualenv
+
+virtualenv venv
+
+echo virtual environment created
+
+venv\Scripts\activate
```
Comment (bartekpacia) [inline install.bat]: probably crashes here, though no errors nor any kind of output is printed

Comment (bartekpacia) [reply install.bat]: Fixed it by adding `CALL`, [this post helped](https://stackoverflow.com/questions/14731877/to-call-or-not-to-call-a-batch-file)

```suggestion
CALL venv\Scripts\activate
```

**Comment by bartekpacia** [comment]
Well, it feels very "rough" and unpolished, but works. I'm open to any suggestions.

**Comment by xprilion** [comment]
Hey @bartekpacia, I'll be going through and testing the script, meanwhile can you please remove the changes in the requirements.txt file? We won't want to merge those accidentally with this PR someday because the PR title/content is not indicative and it might slip

**Comment by bartekpacia** [comment]
Of course @xprilion, done. I committed them by a mistake. [facepalm]

**Comment by devkapilbansal** [comment]
@bartekpacia please link the related issue too in the PR description.

**Comment by devkapilbansal** [comment]
Closing as it is not linked to any issue and the PR is stale

**Pull request closed without merging**